pipeline/pipeline-run.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: component-rbac-gen
+  labels:
```

Comment:
1. should add some labels to indicate which `component`  and `repository` this pipeline is for.
2. should add annotations to indciate this is for `kubebb rating`